gunagala/imager.py

```diff
@@ -1521,9 +1541,6 @@ def get_pixel_coords(self, centre):
         if not isinstance(centre, SkyCoord):
```

Comment:
Should remove the `centre` argument and related code from this method now that there is a `set_WCS_centre` method for setting the centre. Should replace it with a check that CRVAL has been set, and raise an informative error if not (e.g. "CRVAL not set! Must call set_WCS_centre before get_pixel_coords")